src/components/Settings.jsx

```diff
@@ -129,48 +192,15 @@ export default function Settings() {
         <rb.Button
           variant="outline-dark"
           className="border-0 mb-2 d-inline-flex align-items-center"
-          onClick={async (e) => {
+          onClick={(e) => {
             e.preventDefault()
```

Comment:
NIT: I think `e.preventDefault()` can be removed. It doesn't really do anything in this case. I've tested it locally.